src/Components/Nav/Nav.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import SearchModal from '../Modal/SearchModal/SearchModal';
+import Dropdown from '../Dropdown/Dropdown';
+import { Link } from 'react-router-dom';
+import './Nav.scss';
+import LoginModal from '../Modal/LoginModal/LoginModal';
+
+class Nav extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      searchModalValid: false,
+      dropDownValid: false,
+      loginModalValid: false,
+    };
+  }
+  loginModalOn = event => {
+    this.setState({
+      loginModalValid: true,
+    });
+  };
+  loginModalOff = event => {
+    this.setState({
+      loginModalValid: false,
+    });
+  };
+  searchModalOn = event => {
```

Comment:
```suggestion
  constructor() {
    super();
    this.state = {
      searchModalValid: false,
      dropDownValid: false,
      loginModalValid: false,
    };
  }
  
  loginModalOn = event => {
    this.setState({
      loginModalValid: true,
    });
  };
  
  loginModalOff = event => {
    this.setState({
      loginModalValid: false,
    });
  };
  
  searchModalOn = event => {
```
가독성을 위해 함수 정의시 한 칸 씩 띄워주세요~